### DISCUSSION TITLE
Sample portfolionya

### DISCUSSION THREAD

**Discussion by Khanam2023** [deaafrizal/forum#5] [The Lounge]
Bang sharing dong portfolio supaya kami pemula bisa ngikutin portfolio yg bener bang.
Sample portfolionya bang kita kerja di sini ga bisa berkembang bang.
Terima kasih

**Comment by deaafrizal** [comment]
halo bro, btw portofolio yang salah yang kaya gimana emangnya? 😄
selama projeknya bisa di show ke reqruiter pas lu nyari kerja sih harusnya bisa jadi portofolio kok.

kalau sebagai contoh mungkin bisa contek porto repo gue 4 taun lalu berikut ini:
https://github.com/sleeperbox/client
https://github.com/sleeperbox/web-api

atau punya temen gue dengan setup versi taun2 sekarang:
https://github.com/entry-point-community

atau yang di youtube gue, itu juga bisa jadi porto tuh yang playlist semester 3 atau projek2 studi kasus lainnya. 

atau mungkin temen2 lainnya yang liat postingan ini bisa sharing juga 👌

edit: karena gue gak cari kerja lagi, jadi gak ada porto terbaru kecuali apa2 yang gue show di youtube dan repository private dari projekan2 freelance punyanya client / user yang team gue kerjain. semangat terus bray 🙌

**Reply by teguhkrniawan** [reply]
bang, nanya dong. kan ini lu ngasi portofolio dalam bentuk repo ya, dan biasanya kan screening CV pertama kita kan di HR, menurut abang apakah repo ini dipahami HR ? atau better kita membuatnya dalam bentuk gambar ?

**Reply by deaafrizal** [reply]
gak bro, itu cuma repo. aslinya jadi APK. gue kasihnya yang APK apps android dari repo diatas, pas dapet technical interview baru diliat juga repo nya.

**Reply by teguhkrniawan** [reply]
ohh i see, thankyou bang jawabannya.
